addons/project/models/project.py

```diff
@@ -1676,6 +1678,17 @@ def _compute_stage_display(self):
         for task in self:
             task.stage_display = task.stage_id.name if task.project_id else task.personal_stage_id.stage_id.name
 
+    @api.depends('depend_on_ids', 'is_blocked')
+    def _compute_blocking_task_display(self):
+        for record in self:
+            if not record.is_blocked:
```

Comment:
```suggestion
            if not record.is_blocked or task.is_closed:
```
Because we won't use it if it is closed, will we?

Reply:
I actually have a check on the is_closed in the xml but I guess doing this spare some calculation.